index.html

```diff
@@ -212,7 +220,7 @@ <h2 class="about-title">About me</h2>
         </div>
 
         <div class="devider"></div>
-
+        
         <div class="skills">
             <ul class="lang-list">
                 <li><ul class="language list-item1"><li>Language</li></ul></li>
```

Comment:
- [ ] Let's make the contact form look more like the Figma template:

![Captura de pantalla 2022-03-25 a la(s) 8 02 10 p  m](https://user-images.githubusercontent.com/83717285/160219031-a6710ae4-e7fd-440d-8286-b6e0d0cfddde.png)

- Let's start by adding some space on the left side of the text.
- Please align the text to the left, it is currently using a center alignment.
- Kindly remove the big blank space between the text and the form.